CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## [v0.14.1] - unreleased
```

Comment:
In the spirit of semver it'd be better to release as `v0.15.0` since this is a change in behaviour that someone using the gem may/may not want.